core/core.ex

```diff
@@ -59,68 +66,88 @@ defmodule AntikytheraCore do
 
   defp activate_raft_fleet(f) do
     :ok = RaftFleet.activate(AntikytheraEal.ClusterConfiguration.zone_of_this_host())
+
     try do
       f.()
     catch
       type, reason ->
         # When an error occurred in the core part of `start/2`, try to cleanup this node so that
         # existing consensus groups (especially `RaftFleet.Cluster`) are not disturbed by the failing node.
         RaftFleet.deactivate()
-        :timer.sleep(10_000) # wait for a moment in the hope that deactivation succeeds...
+        # wait for a moment in the hope that deactivation succeeds...
+        :timer.sleep(10_000)
         {:error, {type, reason}}
     end
   end
 
   defp start_cowboy_http() do
     dispatch_rules = AntikytheraCore.Handler.CowboyRouting.compiled_routes([], false)
+
     ranch_transport_opts = %{
-      max_connections: :infinity, # limit is imposed on a per-executor pool basis
-      socket_opts:     [port: Antikythera.Env.port_to_listen()],
+      # limit is imposed on a per-executor pool basis
+      max_connections: :infinity,
+      socket_opts: [port: Antikythera.Env.port_to_listen()]
     }
+
     cowboy_proto_opts = %{
       # (a) timeout of a request with no data transfer; must be sufficiently longer than the gear action timeout (10_000)
       # (b) timeout of a connection with no requests; this should be longer than LB's idle timeout
-      idle_timeout:    30_000,  # (a)
-      request_timeout: 120_000, # (b)
-      env:             %{dispatch: dispatch_rules},
-      stream_handlers: [:cowboy_compress_h, :cowboy_stream_h],
+      # (a)
```

Comment:
When formatted, I prefer directly inserting comment like
```suggestion
      # timeout of a request with no data transfer; must be sufficiently longer than the gear action timeout (10_000)
```